### PR TITLE
Feature/feature settings menu

### PR DESCRIPTION
**Pull request by santidrj**

### CODE REVIEW

**Comment by santidrj** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
           

Complexity increasing per file
==============================
- app/src/main/java/org/pesmypetcare/mypetcare/activities/fragments/SettingsMenuFragment.java  2
- app/src/main/java/org/pesmypetcare/mypetcare/activities/fragments/NewPassword.java  3
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care/pullRequest?prid=5213042&bid=17093689)

File: app/src/main/java/org/pesmypetcare/mypetcare/activities/NewPasswordInterface.java [hunk 1]
```diff
@@ -0,0 +1,9 @@
+package org.pesmypetcare.mypetcare.activities;
+
+import androidx.fragment.app.Fragment;
+
+public interface NewPasswordInterface {
+
+    void changeFragmentPass(Fragment fragment);
```
Comment (AlbertPG) [inline app/src/main/java/org/pesmypetcare/mypetcare/activities/NewPasswordInterface.java:10]: The method should be commented

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -13,4 +13,18 @@
     <string name="navigation_view_open">Open</string>
     <string name="navigation_view_closed">Closed</string>
     <string name="not_implemented">This functionality is not implemented</string>
+    <string name="hello_blank_fragment">Hello blank fragment</string>
```
Comment (AlbertPG) [inline app/src/main/res/values/strings.xml]: Why is there a "hello_blank_fragment"?

**Review by AlbertPG — COMMENTED** [review]
Take into account this fixes and make an issue about them

**Review by danicm47 — APPROVED** [review]

**Review by xavier-campos-diaz — APPROVED** [review]

**Review by AlbertPG — APPROVED** [review]
You've got my approval

**Comment by santidrj** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
           

Complexity increasing per file
==============================
- app/src/main/java/org/pesmypetcare/mypetcare/activities/fragments/NewPassword.java  3
- app/src/main/java/org/pesmypetcare/mypetcare/activities/fragments/SettingsMenuFragment.java  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care/pullRequest?prid=5213042&bid=17093689)

**Comment by santidrj** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- app/src/main/java/org/pesmypetcare/mypetcare/activities/fragments/NewPassword.java  3
- app/src/main/java/org/pesmypetcare/mypetcare/activities/fragments/SettingsMenuFragment.java  2
         

Clones added
============
- app/src/main/java/org/pesmypetcare/mypetcare/activities/fragments/NewPassword.java  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care/pullRequest?prid=5213042&bid=17093689)

**Pull request merged**